<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeFraction</class>
 <widget class="QDialog" name="VolumeFraction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Volume Fraction</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Serif&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Grayscale Range of Phase of Interest:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Boundary1Label">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Serif&quot;;</string>
       </property>
       <property name="text">
        <string>Boundary 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Boundary1Edit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="Boundary2Label">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Serif&quot;;</string>
       </property>
       <property name="text">
        <string>Boundary 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Boundary2Edit">
       <property name="text">
        <string>255</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="calculateButton">
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Begin Calculation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Simulation Output</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="porosityLabel">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Serif&quot;;</string>
       </property>
       <property name="text">
        <string>Volume Fraction:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="porositResult">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Boundary1Edit</tabstop>
  <tabstop>Boundary2Edit</tabstop>
  <tabstop>calculateButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
